rosetta/docs/GPU_performance.md

```diff
@@ -52,23 +74,53 @@ For more fine-grained control over which collectives should be asynchronous or n
 
 - --xla_gpu_disable_async_collectives=allreduce,allgather,reducescatter,collectivebroadcast,alltoall,collectivepermute
 
+## Optimization level
+
+- Set via JAX (not an XLA_FLAGS switch):
+```python
+import jax
+jax.config.update("jax_optimization_level", "O1")
+```
+
+- Or via environment variable:
+```
+JAX_OPTIMIZATION_LEVEL=O1 python your_script.py
+```
+
+- Effect details (O1):
+  - Latency Hiding Scheduler (LHS): previously required `--xla_gpu_enable_latency_hiding_scheduler=true`; now enabled by O1. To force explicitly:
+
+    ```
+    --xla_gpu_enable_latency_hiding_scheduler=true
+    ```
+  - Collective pipelining: previously set via flags; now enabled by O1. To force explicitly:
+
+    ```
+    --xla_gpu_enable_pipelined_all_reduce=true
+    --xla_gpu_enable_pipelined_all_gather=true
+    --xla_gpu_enable_pipelined_reduce_scatter=true
+    ```
+  - Unified SOL latency estimator:
+    - Enabled on Hopper/Blackwell at O1 when supported. Single-host (NVLINK) collectives use a perf table; cross-host DCN collectives use the analytical SOL model.
```

Comment:
I think this means within/across NVLink domains (which can be larger than single hosts)? Can you tweak the wording?